collection-scripts/common.sh

```diff
@@ -2,10 +2,19 @@
 
 source "${DIR_NAME}/bg.sh"
 
+export OSP_NS="${OSP_NS-openstack}"
+export OSP_OPERATORS_NS="${OSP_OPERATORS_NS-openstack-operators}"
```

Comment:
+1 this is a good idea. Originally `DEFAULT_NAMESPACES` was a collection of `default` values that might or might not be present in the cluster, and this array was supposed to be grown by passing `ADDITIONAL_NAMESPACES`, but this improves that logic, and I like it.